code/datums/ruins/space.dm

```diff
@@ -182,3 +182,70 @@
+	cost = 5 // This is a chonky boy
+	allow_duplicates = FALSE // This has its own area with an APC, and 2 APCs for the same area breaks badly. Its also huge.
+
+/datum/map_template/ruin/space/clownmime
+	id = "clownmime"
+	suffix = "clownmime.dmm"
+	name = "Clown & Mime Mineral Deposits"
+	description = "A crash site of two opposing factions, both trying to complete mining trips for their own valuable minerals. While all the crew have long perished, the minerals are likely intact."
+
+/datum/map_template/ruin/space/dj
+	id = "dj"
+	suffix = "dj.dmm"
+	name = "Russian DJ Station"
+	description = "An old russian listening station, long since defunct and lifeless, however the equipment is likely still in working condition."
+	cost = 2
+	allow_duplicates = FALSE // This has its own area with an APC, and 2 APCs for the same area breaks badly
+
+/datum/map_template/ruin/space/druglab
+	id = "drublab"
+	suffix = "druglab.dmm"
+	name = "Drug Lab"
+	description = "An old abandoned \"Chemistry\" site, which has a strong aura of amphetamines around it."
+	allow_duplicates = FALSE // This has its own area with an APC, and 2 APCs for the same area breaks badly
+
+/datum/map_template/ruin/space/syndiedepot
+	id = "syndiedepot"
+	suffix = "syndiedepot.dmm"
+	name = "Suspicious Supply Depot"
+	description = "A syndicate supply depot, heavily stocked, but heavily guarded with an assortment of shields, sentry bots, armed operatives and more."
+	allow_duplicates = FALSE // One of these is overpowered enough
+	cost = 5 // This is one of the biggest ruins on the server, so it has a high cost, not to mention the amount of loot you can get out of it
+
+/datum/map_template/ruin/space/ussp_tele
+	id = "ussp_tele"
+	suffix = "ussp_tele.dmm"
+	name = "USSP Teleporter"
+	description = "An old, almost fully destroyed teleporter, seemingly part of what used to be a much larger structure."
+	allow_duplicates = FALSE // This has its own area with an APC, and 2 APCs for the same area breaks badly
+
+/datum/map_template/ruin/space/ussp
+	id = "ussp"
+	suffix = "ussp.dmm"
+	name = "USSP"
+	description = "A decript station of seemingly russian origin. The last contact had with this station was a distress signal, and the rest was dark."
+	allow_duplicates = FALSE // One of these has enough loot
+	cost = 5 // This ruin is 100x100 tiles, so we dont want it to be treated like a 10x10 meteor
+
+/datum/map_template/ruin/space/whiteship
+	id = "whiteship"
+	suffix = "whiteship.dmm"
+	name = "NT Medical Ship"
+	description = "An old, abandoned NT medical ship. Its computer can navigate to other landmarks within space with ease."
+	allow_duplicates = FALSE // I dont even want to think about what happens if you have 2 shuttles with the same ID. Likely scary stuff.
+	always_place = TRUE // Its designed to make exploring other space ruins more accessible
```

Comment:
since this still has a cost but is guaranteed, the effective budget is 19 to 29, right?

Reply:
White ship doesnt have a cost associated with it, making it `1`, not `10`.

Reply:
...yes? And since the budget seems to be rand(20, 30), that means your effective budget after spawning the guaranteed white ship is actually 19 to 29.

Reply:
All always_spawn ruins should probably have assumed cost=0. Otherwise you have to manually subtract them every time to find out the real ruin budget.

Reply:
Oh wait I misinterpreted what was said.